debian/copyright

```diff
@@ -201,15 +201,55 @@ License: OFL-1.1
 
 Files: src/tools/rest-frontend/public/vendor/asciinema/asciinema-player.*
 Copyright: 2011-2016 Marcin Kulik
-License: Apache-2.0
+License: GPL-3-asciinema
```

Comment:
The license was changed from GPL-3 to Apache-2.0 in [this commit from June 2017](https://github.com/asciinema/asciinema-player/commit/8b24d8dcefe1f7419d06bc5185c4896ca51aff12) but according to our history the files were added before. So technically they were GPL-3 then.

I could not determine whether it is/was [GPL-3](https://spdx.org/licenses/GPL-3.0-only.html) or [GPL-3+](https://spdx.org/licenses/GPL-3.0-or-later.html). The only difference between those licenses I could find seems to be in the license header within files. Since the player's source files do not contain such a header I assumed it was GPL-3 only.

Reply:
This would be a problem, we could not distribute it within our repo then. Can you please replace the file to the one that uses Apache-2.0?

Reply:
Oh I see. It's, done - PR #1804 has the necessary changes.